server/s3_motr_rw_common.cc

```diff
@@ -195,6 +195,53 @@ void s3_motr_op_stable(struct m0_op *op) {
   s3_log(S3_LOG_DEBUG, request_id, "%s Exit", __func__);
 }
 
+// Motr callbacks, run in motr thread
+void s3_motr_op_executed(struct m0_op *op) {
+  s3_log(S3_LOG_DEBUG, "", "%s Entry\n", __func__);
+  struct s3_motr_context_obj *ctx = (struct s3_motr_context_obj *)op->op_datum;
+
+  S3AsyncOpContextBase *app_ctx =
```

Comment:
This code below is an exact copy from op_stable().  Create separate function with common code, and call it from both op_executed and op_stable.